test/run_pass/test_net_http_request_response.js

```diff
@@ -23,7 +23,7 @@ var message = 'Hello IoT.js';
 // Options for further requests.
 var options = {
   method: 'POST',
-  port: 3005,
+  port: 12306,
```

Comment:
I think it's better to introduce the PORT in `common.js`, see https://github.com/nodejs/node/blob/master/test/common/index.js#L794-L799.